src/commands/import.ts

```diff
@@ -92,6 +92,20 @@ export default class Import extends Kommand {
         this.logKo(`Error during import of ${file}: ${error.message}. Skipped.`)
       }
     }
+    for (const file of files.usersMappings) {
+      try {
+        this.logInfo(`[users] Start importing users mappings in ${file}`)
+        const dump = JSON.parse(fs.readFileSync(file, 'utf8'))
+
+        const mapping = dump.content.mapping
+        await this.sdk?.security.updateUserMapping({ properties: mapping })
+
+        this.logOk('[users] mappings imported')
```

Comment:
Again, why would this be in plural. Are there any differences with these:
https://github.com/kuzzleio/kourou/blob/e180dd491540502315f642c3c0586140b3324901/src/commands/import.ts#L61
https://github.com/kuzzleio/kourou/blob/c3695994abfd264ae05cc2143ca8770bf7e3c41d/src/commands/document/create.ts#L57